pkg/parse/quote_test.go

```diff
@@ -24,8 +24,8 @@ func TestQuote(t *testing.T) {
 
 		// Double quote when there is unprintable char.
 		Args("a\nb").Rets(`"a\nb"`),
-		Args("\x1b\"\\").Rets(`"\e\"\\"`),
-		Args("\x00").Rets(`"\x00"`),
+		Args("\000\x1b\"\\").Rets(`"\x00\e\"\\"`),
```

Comment:
The NUL byte is already tested in the test case below, why introduce it to this test case too?

Reply:
Good point. The intent was to verify that both forms (`"\000"` and `"\u0000"`) where handled identically by Elvish. But, obviously in hindsight, this is actually verifying that Go translates both sequences to a NUL byte. I'll remove the second, redundant, test. This does, however, raise the question of how to verify Elvish, rather than Go, interprets such sequences. Presumably there is Elvish test code that does so but I couldn't find it.